pwem/protocols/protocol_movies.py

```diff
@@ -116,7 +116,7 @@ def _insertAllSteps(self):
 
     # STEP to convert correction images if apply
     def _convertInputStep(self):
-
+        pwutils.makePath(self._getExtraPath('DONE'))
```

Comment:
My only concern is if this is a right place to do this.

Reply:
Yeah! I understood the reason. Another way this could be done, freeing up the "when" is something like:

def getDONEFile(self, *paths):

 if self._doneFolder is None:
    self._doneFolder = self._getExtra("DONE")
    if not os.path.exists(self._doneFolder):
        pwutils.makePath(self._doneFolder)

 return os.path.join(self._doneFolder, *paths)

This way, "DONE" is only in one place. And creation happens only once the first time is called. In exchange, there is always an "if to evaluate".

Reply:
Hopefully we will avoid all these done files soon.